opentelemetry-exporters-datadog.gemspec

```diff
@@ -26,6 +29,7 @@ Gem::Specification.new do |spec|
 
   spec.homepage    = 'https://github.com/Datadog/dd-opentelemetry-exporter-ruby'
   spec.license     = 'Apache-2.0'
+  spec.metadata    = { "otlp_ingest_in_datadog_agent" => "https://docs.datadoghq.com/tracing/setup_overview/open_standards/#otlp-ingest-in-datadog-agent" }
```

Comment:
what does this do? how does it get rendered?

Reply:
https://guides.rubygems.org/specification-reference/#metadata . Docs say "These links will be used on your gem’s page on rubygems.org"; I guess they'll show up under links in <https://rubygems.org/gems/opentelemetry-exporters-datadog>?